src/comet_llm/logging.py

```diff
@@ -15,10 +15,12 @@
 import logging
 import sys
 
+from . import config
+
 
 def setup() -> None:
     root = logging.getLogger("comet_llm")
-    root.setLevel(logging.INFO)
+    root.setLevel(config.logging_level())
```

Comment:
You need to add check for `None` here because `config.logging_level()` returns `Optional[str]`. You must be sure that returned value is not `None` before setting it as log level.